locations/spiders/carrefour_fr.py

```diff
@@ -1,6 +1,40 @@
 from locations.categories import Categories, apply_category
+from locations.items import Feature
 from locations.storefinders.woosmap import WoosmapSpider
 
+CARREFOUR_SUPERMARKET = {
+    "brand": "Carrefour",
+    "brand_wikidata": "Q217599",
+    "extras": Categories.SHOP_SUPERMARKET,
```

Comment:
`.value` is needed as a suffix when using categories in this way.

Reply:
I'm using `apply_category()`. The name of the property in brand mappings is misleading, sorry for that, changed it to `category`.